silverbackis/api-component-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Silverback\\ApiComponentBundle\\SilverbackApiComponentBundle": ["all"],
+        "Cocur\\Slugify\\Bridge\\Symfony\\CocurSlugifyBundle": ["all"],
+        "Liip\\ImagineBundle\\LiipImagineBundle": ["all"]
```

Comment:
You should remove everything that is not `common-js-bundle`. Not add more bundles here =)

Reply:
Ah! I had not seen the previous comment before adding this

I'll do that :)